lib/dolly/request.rb

```diff
@@ -80,6 +80,10 @@ def request method, resource, data = nil
       end
     end
 
+    def replicate! target_db, opts={}
+      Dolly::SimpleReplicator.new(self, target_db, opts).replicate!
```

Comment:
🎉

Reply:
This cant work now that I think about it. simple replicator requires request which must be required by this in order to call it as a method.  I think I will have to make another separate object for this. `Dolly::Replication.new` seems appropriate.

Reply:
this can have a `run!` method.